src/dataSource.ts

```diff
@@ -11,13 +13,15 @@ if (process.env.NODE_ENV === 'test') {
 } else {
   dataSource = new DataSource({
     type: 'postgres',
-    host: Config.database.host,
-    port: Config.database.port,
-    username: Config.database.username,
-    password: Config.database.password,
-    database: Config.database.name,
-    synchronize: false,
-    logging: false,
+  host: Config.database.host,
```

Comment:
Why the indentation changes?
Make sure you always lint your code before pushing.

Reply:
because we have the same lines in the code.

Reply:
Yeah I know but why do they moved backwards?
Try `linting` your code before push.

Reply:
I think it is because of the conflict I resolved.